test/utils/noderesourcetopologies/noderesourcetopologies.go

```diff
@@ -434,13 +465,9 @@ func ResourceInfoProviding(resources []nrtv1alpha2.ResourceInfo, resName string,
 	return true
 }
 
-func FilterByPolicies(list []nrtv1alpha2.NodeResourceTopology, policies []nrtv1alpha2.TopologyManagerPolicy) []nrtv1alpha2.NodeResourceTopology {
-	var filteredNrts []nrtv1alpha2.NodeResourceTopology
-	for _, policy := range policies {
-		nrts := FilterTopologyManagerPolicy(list, policy)
-		filteredNrts = append(filteredNrts, nrts...)
-	}
-	return filteredNrts
+// REVIEW - Could avoid this call
```

Comment:
In general I'm against trivial wrappers. I'd just update the call sites to use `FilterByTopologyManagerPolicy`